packages/yourloops/pages/password-reset/request-password-reset.tsx

```diff
@@ -37,26 +35,31 @@ import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
 import Container from "@material-ui/core/Container";
 import Grid from "@material-ui/core/Grid";
+import { RouteComponentProps } from "react-router-dom";
 import TextField from "@material-ui/core/TextField";
 import Typography from "@material-ui/core/Typography";
-
-import { t } from "../../lib/language";
+import _ from "lodash";
 import brandingLogo from "branding/logo.png";
-import { useState } from "react";
 import { useAuth } from "../../lib/auth/hook/use-auth";
+import { useState } from "react";
```

Comment:
```suggestion
```

Reply:
That's why I don't like `import * as React from "react";` This makes the code much heavier.

Reply:
For me writing `React.useState()` is cleaner anyway 😉

Reply:
I don't get why, but as you wish 😄